go/cmd/vtctldclient/command/schema.go

```diff
@@ -137,15 +137,14 @@ func commandApplySchema(cmd *cobra.Command, args []string) error {
 	ks := cmd.Flags().Arg(0)
 
 	resp, err := client.ApplySchema(commandCtx, &vtctldatapb.ApplySchemaRequest{
-		Keyspace:                ks,
-		AllowLongUnavailability: applySchemaOptions.AllowLongUnavailability,
```

Comment:
Are we deprecating it or removing it? The title says deprecate but this is removal isn't it, since any value provided by the user will be ignored. No?

Or maybe it's simply no longer needed so we're keeping the flag around and marking it as deprecated so that it gives users a release to remove the flag in their scripts etc? I will assume that is the case.

Reply:
> Or maybe it's simply no longer needed so we're keeping the flag around and marking it as deprecated so that it gives users a release to remove the flag in their scripts etc? I will assume that is the case (the description also seems to be saying this).

That's the one. We remove the functionality, but keep the flag so that `vttablet` execution does not break _yet_.